apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1692,12 +1696,16 @@ validate_prev_key_block(_MicroHeader, not_found, _PrevKeyHeader) ->
 
 validate_micro_signature(MicroHeader, _PrevHeader, PrevKeyHeader)
   when PrevKeyHeader =/= not_found ->
-    Bin = aec_headers:serialize_to_signature_binary(MicroHeader),
-    Sig = aec_headers:signature(MicroHeader),
-    Signer = aec_headers:miner(PrevKeyHeader),
-    case enacl:sign_verify_detached(Sig, Bin, Signer) of
-        true  -> ok;
-        false -> {error, signature_verification_failed}
+    case aec_headers:consensus_module(MicroHeader) of
+        aec_consensus_hc -> ok;
```

Comment:
Here I would like a little comment on why we don't need to validate the signature here.
I guess this is because the leader of the epoch is known and all validators will reject it if the leader has not signed... but where is that done?